WordPress/src/main/res/values/stats_styles.xml

```diff
@@ -85,7 +85,7 @@
 
     <style name="JetpackTermsAndConditionsButton">
         <item name="android:autoLink">all</item>
-        <item name="android:linksClickable">false</item>
+        <item name="android:linksClickable">false</item>4f748e
```

Comment:
Probably some misplaced paste?
```suggestion
        <item name="android:linksClickable">false</item>
```

Reply:
Interesting that the build doesn't fail :-)